src/evict/evict_conn.c

```diff
@@ -222,6 +222,9 @@ __wt_evict_config(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
     WT_RET(__wt_config_gets(session, cfg, "eviction.evict_sample_inmem", &cval));
     conn->evict_sample_inmem = cval.val != 0;
 
+    WT_RET(__wt_config_gets(session, cfg, "eviction.evict_use_npos", &cval));
+    conn->evict_use_npos = cval.val != 0;
```

Comment:
Should we use atomic store here?

Reply:
WT doesn't consistently use `wt_shared` and I followed the convention.
But I agree that it's good to use it, so I added it.
Thanks for noticing!